projects/output-and-error-handling/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+func handleWeatherRequest() (string, error) {
+
+	resp, err := http.Get("http://localhost:8080")
+
+	if err != nil {
+
+		if os.IsTimeout(err) {
+			fmt.Fprintf(os.Stderr ,"request timed out.\n");
```

Comment:
You're not actually formatting here, but do include a trailing newline, so may want to write this as:
```suggestion
			fmt.Fprintln(os.Stderr ,"request timed out.");
```